frontend/src/app/components/backlog-content/backlog-content.component.ts

```diff
@@ -205,7 +206,26 @@ export class BacklogContentComponent implements OnInit {
     });
   }
 
+  ngOnChanges(): void {
+    this.filterItems();
+  }
+
   ngOnInit(): void {
+    if (this.currentUser === undefined) {
+      this.role = 0;
+    } else {
+      this.role =
+        (this.currentUser?.organizationRoles?.find(
+          (m) =>
+            m.organizationId === this.project.organizationId &&
+            m.userId === this.currentUser.id,
+        )?.role as UserRole) || 0;
+
+      if (UserRole.OrganizationAdmin === this.role) {
```

Comment:
Also we have not only Organization admin, but project admin, so I think we can enhance this by defining this variable as true if current User role more or equal than project Admin